app/js/functions.js

```diff
@@ -193,12 +193,7 @@ function parseHash(fieldsById) {
   }
 
 //Inital map setup
-var map = d3.select("#map"),
```

Comment:
When I removed the others, it broke the map :(

Removing just that one kept the map running, saved on an operation. just stuff I found out while figuring out the d3 and map.